README.md

```diff
@@ -1,2 +1,81 @@
 # node-axosoft
 A node.js client for the Axosoft API
+
+This library is built to support v6 of the Axosoft REST API and has been tested with both axosoft.com hosted accounts and Axosoft installed on premise.  Any questions can be sent to api@axosoft.com.
+
+[Here's](http://developer.axosoft.com/) a link to our full documentation for the Axosoft REST API.
+
+## Installation
+
+  Install with the node package manager [npm](http://npmjs.org):
+
+    $ npm install node-axosoft
+
+## Examples
+
+### Create the Axosoft Connection
+
+    const nodeAxosoft = require('node-axosoft');
+    const axosoftUrl = {Axosoft Url};
```

Comment:
I understand what this is but it looks a bit awkward with an actual object declaration right below it. Maybe just `https://example.axosoft.com/`?